src/cross_module_fn.c

```diff
@@ -160,6 +160,13 @@ process_compress_table_default(AlterTableCmd *cmd, Hypertable *ht,
 	pg_unreachable();
 }
 
+static void
+process_compress_table_add_column_default(int32 htid, int32 compress_htid, ColumnDef *orig_def)
```

Comment:
added a new altercmd processing function to tsl/src/utility.c. This functionality has been moved there.

Reply:
Sorry, I didn't mean to just move the function to `tsl/process_utility.c`; I meant that we do not need a cross-module function since we already forward ddl commands to TSL. If we add a cross-module function definition for every DDL command, including every alter-table variant, the cross-module interface is going to blow-up with a ton of functions.

Thus, unless I am missing something, you should be able to "capture" the alter table in TSL's process utility instead of capturing it outside TSL and then forwarding it to TSL via a cross-module function.